MixItUp.WPF/Controls/Command/ActionGroupCommandDetailsControl.xaml.cs

```diff
@@ -58,6 +59,7 @@ public override Task Initialize()
                     this.command.Name = this.NameTextBox.Text;
                 }
                 this.command.Unlocked = this.UnlockedControl.Unlocked;
+                this.command.IsRandomized = this.RunOneRandomlyToggleButton.IsChecked ?? false;
```

Comment:
You can actually just add ".GetValueOrDefault()" to the end of ".IsChecked", that'll just get you the the right value or false.